themes/_default.yaml

```diff
@@ -1,5 +1,7 @@
 generic:
   border:
+    radius-default-btn: &radius-default-btn 2px
+    radius-ghost-btn: &radius-ghost-btn 2px
```

Comment:
I was looking for the usage of those. Could you point me out where you're using this?

Reply:
![download](https://user-images.githubusercontent.com/16314636/31891751-9f66f650-b807-11e7-8401-90e4bc3dfc4c.png)